live-examples/html-examples/text-content/css/dl.css

```diff
@@ -0,0 +1,14 @@
+@font-face {
+    font-family: "Fira Sans";
+    src: local("FiraSans-Regular"),
+    url("/media/fonts/FiraSans-Regular.woff2") format("woff2");
+}
+
+p, dl {
```

Comment:
Nit: Code style: Each selector on its own row

Reply:
This line was copied from another example (`<ol>`), so that one should be updated too.

Reply:
Ignore this push. Something very weird went wrong, and the wrong stuff went up.